File: src/dotnet/EntityFX.NetBenchamarks.Core/Generic/CallBenchmark.cs
using System;
using System.Diagnostics;
using System.Linq;
using System.Runtime.CompilerServices;

namespace EntityFX.NetBenchmark.Core.Generic
{

    public class CallBenchmark : CallBenchmarkBase<float>, IBenchamrk
    {

        public override float BenchImplementation()
        {
            return 0.0f;
        }

        public override BenchResult Bench()
        {
            BeforeBench();
            var sw = new Stopwatch();
            sw.Start();

            var callTime = DoCallBench();


            var result = PopulateResult(BuildResult(sw), callTime);
            result.Elapsed = TimeSpan.FromMilliseconds(callTime);
            DoOutput(result);
            AfterBench(result);
            return result;
        }
    }
}

File: src/python/entityfx/call_benchmark.py
from entityfx.benchmark_base import BenchmarkBase
from entityfx.call_benchmark_base import CallBenchmarkBase
import time

class CallBenchmark(CallBenchmarkBase):
    
    def benchImplementation(self) -> float:
        return .0
    
    def bench(self) :
        self._beforeBench()
        start = time.time()
        call_time = self._doCallBench()
        result = self.populateResult(self._buildResult(start), call_time)
        result["Elapsed"] = time.time() - start
        self._doOutput(result)
        self._afterBench(result)
        return result